mobile/libuplink_android/app/src/androidTest/java/io/storj/mobile/libuplink/LibuplinkInstrumentedTest.java

```diff
@@ -7,26 +7,42 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import java.io.BufferedOutputStream;
+import java.io.BufferedWriter;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+import io.storj.libuplink.mobile.Bucket;
```

Comment:
`import io.storj.libuplink.mobile.*;`

Reply:
its how IDE managing imports in Java. It will move to `io.storj.libuplink.mobile.*` when you will cross a configured number of common imports.